Jucer2Reprojucer/main.cpp

```diff
@@ -498,19 +498,52 @@ int main(int argc, char* argv[])
     if (projectType == "audioplug")
     {
       wLn("jucer_audio_plugin_settings(");
-      convertOnOffSettingWithDefault(jucerProject, "buildVST", "BUILD_VST", true);
-      convertOnOffSettingWithDefault(jucerProject, "buildVST3", "BUILD_VST3", false);
-      convertOnOffSettingWithDefault(jucerProject, "buildAU", "BUILD_AUDIOUNIT", true);
-      convertOnOffSettingWithDefault(jucerProject, "buildAUv3", "BUILD_AUDIOUNIT_V3",
-                                     false);
-      convertOnOffSettingWithDefault(jucerProject, "buildRTAS", "BUILD_RTAS", false);
-      convertOnOffSettingWithDefault(jucerProject, "buildAAX", "BUILD_AAX", false);
-      if (jucerVersionAsTuple >= Version{5, 0, 0})
+
+      if (jucerVersionAsTuple >= Version{5, 3, 1})
       {
-        convertOnOffSettingWithDefault(jucerProject, "buildStandalone",
-                                       "BUILD_STANDALONE_PLUGIN", false);
-        convertOnOffSettingWithDefault(jucerProject, "enableIAA", "ENABLE_INTERAPP_AUDIO",
+        const auto convertIdsToStrings =
+          [](const juce::var& v,
+             const std::map<juce::String, const char*>& idsToStrings) {
+            const auto ids = juce::StringArray::fromTokens(v.toString(), ",", {});
+            juce::StringArray strings;
+            for (const auto& idToString : idsToStrings)
```

Comment:
replace with cpp17 structured binding for even better readability ? (fromhttps://skebanga.github.io/structured-bindings/ )
```cpp
for (const auto&& [ id, string ] : idsToStrings)
{
    if (ids.contains(id))
              {
                strings.add(string);
              }
}
```

Reply:
This version is way better than the 1st proposal. maybe a cpp17 feature is not a necessary constraint for FRUT, therefore I'm accepting the changes. feel free to include or not this suggestion.